OpenOversight/test_data.py

```diff
@@ -9,22 +9,22 @@
 
 def populate():
     """ Populate database with test data"""
-    po1 = models.Officer(last_name='Tinkle', first_name='Ivana', race='BLACK',
+    po1 = models.Officer(last_name='TINKLE', first_name='IVANA', race='BLACK',
```

Comment:
You can image there are officers who have mixed cased last names (i.e. `McAuliffe`)